modules/core/src/test/scala/ru/tinkoff/phobos/decoding/AttributeDecoderTest.scala

```diff
@@ -0,0 +1,23 @@
+package ru.tinkoff.phobos.decoding
+
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+import ru.tinkoff.phobos.annotations.XmlCodec
+import ru.tinkoff.phobos.syntax.attr
+
+import java.time.OffsetDateTime
+
+class AttributeDecoderTest extends AnyWordSpec with Matchers {
```

Comment:
Good Idea to add these tests, I should implement cases for other primitive instances